<launch>
    <!-- RViz with shadow robot -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find hb_grasp_generator)/config/test.rviz" />

    <!-- Verbose mode -->
    <arg name="verbose" default="false" />
    <arg unless="$(arg verbose)" name="command_args" value="" />
    <arg     if="$(arg verbose)" name="command_args" value="--verbose" />

    <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
    <include file="$(find bender_moveit_config)/launch/planning_context.launch">
        <arg name="load_robot_description" value="true"/>
    </include>

    <!-- Publish world frame -->
    <node pkg="tf" type="static_transform_publisher" name="tf_publisher_world" args="0 0 0 0 0 0 /world /bender/base_link 25" />

    <!-- Grasp filter node -->
    <node pkg="hb_grasp_generator" type="grasp_filter_test" name="grasp_filter_test" output="screen" >
        <rosparam command="load" file="$(find hb_grasp_generator)/config/grasp_generator.yaml" />
    </node>

</launch>
